regcore_write/views/document.py

```diff
@@ -26,8 +26,7 @@
             'items': {'type': 'string'}
         },
         'title': {'type': 'string'},
-        'node_type': {'type': 'string'},
-        'preorder_idx': {'type': 'integer', 'minimum': 0}
+        'node_type': {'type': 'string'}
```

Comment:
Sorry to miss this earlier -- we'll need to either keep the previous `preorder_idx` attr in this schema or remove it from the parser's output. Otherwise we won't be able to write new documents.